lib/elixir/lib/io/ansi.ex

```diff
@@ -42,6 +42,12 @@ defmodule IO.ANSI do
     Application.get_env(:elixir, :ansi_enabled, false)
   end
 
+  @doc "Sets foreground color"
```

Comment:
@spec color(0..255) :: String.t
not sure if result value should be `ansidata` instead of `String.t`